static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -17,6 +17,7 @@ export const errorConfig: IssueCategoryConfigMapping = {
       deleteAndDiscard: {enabled: true},
       ignore: {enabled: true},
       merge: {enabled: true},
+      resolve: {enabled: true},
```

Comment:
is it really necessary to add `enabled:true` to all of the issue types if the base type by default has it enabled? maybe i am not understanding typescript correctly

Reply:
typescript gets mad at me if I remove it - I think the default might be for the 'actions' as a whole